specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/deviceupdate.json

```diff
@@ -682,6 +746,33 @@
         }
       }
     },
+    "Identity": {
```

Comment:
this is available in common-types, any reason to not re-use that?

Reply:
The common type includes identity type enums we do not support at the moment like "UserAssigned" and "SystemAssigned, UserAssigned"

Reply:
Also I think the common definition did no include "None" which is a bug.

Reply:
Both common types v1 and v2 contain single enum value: SystemAssigned
See https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json#L238

I can submit a PR to extend that. What should be the values?

```
None
SystemAssigned
UserAssigned
SystemAssigned, UserAssigned
```

Or how Swagger support multiple values?

Reply:
please leave as-is for now, we are discussing making the update to the common-types files to support UserAssigned and None